docs/book/installation/installation.rst

```diff
@@ -49,7 +47,7 @@ This package has the whole ``sylius/sylius`` package in vendors, so you can easi
 .. warning::
 
     During the ``sylius:install`` command you will be asked to provide important information, but also its execution ensures
-    that the default **currency** (USD) and the default **locale** (English - US) are set they can be later on changed in the ``parameters.yml`` file.
+    that the default **currency** (USD) and the default **locale** (English - US) are set they can be later on changed.
```

Comment:
they're now stored in the `config/services.yaml` file, I think. Am I right @pamil ?

Reply:
On my local fresh install there is only the `locale` (as in any Symfony standard), but I cannot find the `currency` parameter.

Btw, the install seems to currently be broken (assets are not installed correctly).

Reply:
what happens if you add the `currency: EUR` below the locale?

Reply:
Edit: It works :)
Check via `bin/console debug:container --parameters | grep currency`

Reply:
It may work, but the currency is set somewhere (I chosen `EUR` during the install, and this currency is indeed used).

Reply:
afaik we don't use a `currency` parameters for last two years or something :D we have now base currencies on channels

Reply:
It looks like this config is now stored in the DB (in the `sylius_currency` table). Am I right?

Reply:
@CoderMaggie there is no such parameter in the container of my app.

Reply:
@dunglas there is no such parameter [since Nov 2016](https://github.com/Sylius/Sylius/pull/6901/commits/29bf4cfb1c220f6f192b1fb438d86ef2b863980e), it's a documentation residue, that should be removed for sure

Reply:
Got it, but the installer still ask for the locale. It may worth it to explain how to change it afterward. In the admin panel? (Sorry I’m a total newcomer to Sylius).

Reply:
At least one currency is required for Sylius to work, it's then set as a base currency of a channel, that is also created during the installation process. No problem, asking questions is good :)

Reply:
And I totally agree it should be explained in installation docs, instead of this legacy information about parameters 👍

Reply:
:+1:

Reply:
Updated